docs/modules/td3.rst

```diff
@@ -14,7 +14,7 @@ We recommend reading `OpenAI Spinning guide on TD3 <https://spinningup.openai.co
 
 .. warning::
 
-  The TD3 model does not support ``stable_baselines3.ppo.policies`` because it uses double q-values
+  The TD3 model does not support ``stable_baselines3.common.policies`` because it uses double q-values
```

Comment:
about that, I'm not sure if we should still keep those "common" policies as the default (because they don't work with 50% of the algorithm now). That's also why I try to use strings in the example to refer to a policy.

Reply:
I am not sure if I understand your point here.

Reply:
in SB2, it was called `common.policies.MlpPolicy`because it was used by most algorithms (A2C, ACER, ACKTR, PPO, TRPO).
However, now, this is not the case anymore

Reply:
Ah, right, I understand now. Perhaps if we move ActorCriticPolicies to a new file, e.g. `common/actor_critic_policy.py`? I would like to avoid the cross-algorithm imports so they do not interfere with each other and it is clear what is used by what :)

Reply:
I understand your point. Maybe, keep it in `policies.py` but we should always use `from stable_baselines3.ppo import MlpPolicy` in the doc
(and anyway, in the doc, I try to use strings as it is the best practice not to fuck up things)

Reply:
Now I feel your point: Importing algorithm-specific policies from "[algorithm_name].policies" translates well to any algorithm, rather than knowing that A2C/PPO need "common.policies", TD3 needs "td3.policies" etc. How about having "policies.py" for each algorithm, but they can then import a shared policy from common, i.e. only contain lines like

```python
# ppo/policies.py file
from stable_baselines3.common.policies import OnlineActorCriticPolicy, OnlineActorCriticCnnPolicy
```

Reply:
it was my point ;) (just afraid we have to tweak the auto doc)